cmd/external-plugins/cherrypicker/server_test.go

```diff
@@ -481,6 +488,9 @@ func testCherryPickPR(clients localgit.Clients, t *testing.T) {
 	var expectedFn = func(branch string) string {
 		expectedTitle := fmt.Sprintf("[%s] This is a fix for Y", branch)
 		expectedBody := fmt.Sprintf("This is an automated cherry-pick of #%d", prNumber)
+		if branch == "release-1.3" {
```

Comment:
I think the new functionality in this PR should support long chains transparently, can you add a test for that?